Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerRmiController.java

```diff
@@ -57,6 +90,11 @@ public void sessionCreated(String roomName, String hostName, boolean hasPassword
         }
     }
 
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)